scripts/fast-reboot

```diff
@@ -514,6 +516,19 @@ function check_docker_exec()
     done
 }
 
+function check_pfc_storm_active()
+{
+    # check if PFC storm is active (abort warmboot) or restored (proceed with warmboot)
+    PFC_STORMED="stormed"
+    pfcwd show stats | grep -i stormed || PFC_STORMED="not_stormed"
```

Comment:
Can you check if this value can be obtained from DB? Depending on grep is dangerous, particularly when it is not strict.